@xen-orchestra/web-core/lib/components/info/UiInfo.vue

```diff
@@ -0,0 +1,56 @@
+<!-- v2 -->
+<template>
+  <div class="vts-ui-info">
+    <UiIcon class="icon" :color :icon />
+    <p class="message"><slot /></p>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import type { IconColor } from '@core/types/color.type'
+import {
+  faCheckCircle,
+  faExclamationCircle,
+  faInfoCircle,
+  faXmarkCircle,
+  type IconDefinition,
+} from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+type Props = {
+  status: 'info' | 'success' | 'warning' | 'danger'
```

Comment:
To maintain consistency with the last naming guidelines, I think `accent` would be more suitable than `status`, and `info` could be renamed as `brand`.
```suggestion
  accent: 'brand' | 'success' | 'warning' | 'danger'
```